daemon/api_systems.go

```diff
@@ -99,6 +103,59 @@ func getSystems(c *Command, r *http.Request, user *auth.UserState) Response {
 	return SyncResponse(&rsp)
 }
 
+type oneSystemResponse struct {
```

Comment:
Super nitpick: It's more common to use `single` instead of `one`  (But feel free to ignore)

Reply:
Nice suggestion - I will also do this in followup discussed above :)